src/main/java/com/faforever/client/player/PlayerInfoWindowController.java

```diff
@@ -198,6 +198,15 @@ private void setAvailableAchievements(List<AchievementDefinition> achievementDef
   }
 
   public void setPlayer(PlayerBean player) {
+    if (player.getLeaderboardRatings().isEmpty()) {
+      log.info("no ratings found");
```

Comment:
This seems more like a debug than info. It also isn't very informative.  I would at least add the name of the player or something to tell us what is causing it or from where. But if this is expected then I am not sure if it is worth logging.